### PR TITLE
*: use fs::{read, read_to_string, write} where possible

### PR DESCRIPTION
**Pull request by kennytm**
## What have you changed? (mandatory)

Refactor some existing code to use the standard `fs::{read, read_to_string, write}` functions if possible.

## What are the type of the changes? (mandatory)

- Engineering (engineering change which doesn't change any feature or fix any issue)

## How has this PR been tested? (mandatory)

Run `cargo build` and `cargo test --no-run`.

## Does this PR affect documentation (docs) update? (mandatory)

No

## Does this PR affect tidb-ansible update? (mandatory)

No

### CODE REVIEW

**Comment by siddontang** [comment]
PTAL @zhangjinpeng1987

**Comment by kennytm** [comment]
The error in https://internal.pingcap.net/idc-jenkins/blue/organizations/jenkins/tikv_ghpr_integration_common_test/detail/tikv_ghpr_integration_common_test/206/pipeline seems like a floating-point error of the randgen test itself. Gonna retry.

/run-integration-common-test

**Comment by siddontang** [comment]
> seems like a floating-point error of the randgen test itself.

Through the output
```
2019/04/03 00:32:36.684 main.go:645: [fatal] run test [udaf_sum_union] err: sql:select sum(`col_float_signed`) as a from (select * from `table20_int_autoinc` union all select * from `table60_int_autoinc`) tmp order by a;: failed to run query 

"select sum(`col_float_signed`) as a from (select * from `table20_int_autoinc` union all select * from `table60_int_autoinc`) tmp order by a;" 

 around line 553, 

we need(163):

select sum(`col_float_signed`) as a from (select * from `table20_int_autoinc` union all select * from `table60_int_autoinc`) tmp order by a;

a

-22318.25227888135

s

but got(163):

select sum(`col_float_signed`) as a from (select * from `table20_int_autoinc` union all select * from `table60_int_autoinc`) tmp order by a;

a

-22318.252278881373
```

I think we may have other problems.

File: src/util/file.rs [hunk 1]
```diff
@@ -18,10 +18,7 @@ use std::path::Path;
 use crc::crc32::{self, Digest, Hasher32};
 
 pub fn read_all<P: AsRef<Path>>(path: P) -> io::Result<Vec<u8>> {
```
Comment (BusyJay) [inline src/util/file.rs]: Seems the method can be removed.

Comment (kennytm) [reply src/util/file.rs]: Turns out this is only used in `src/util/security.rs`. I'll replace it.

**Comment by zhangjinpeng1987** [comment]
@kennytm Please don't force push, this will may incremental review impossible.

**Comment by kennytm** [comment]
@zhangjinpeng1987 That force-push was just for the latest commit which I forgot to sign-off 😝

**Comment by kennytm** [comment]
PTAL @overvenus @siddontang

**Pull request merged**